contracts/Vault.vy

```diff
@@ -290,6 +291,7 @@ def initialize(
     self.performanceFee = 1000  # 10% of yield (per Strategy)
     log UpdatePerformanceFee(convert(1000, uint256))
     self.managementFee = 200  # 2% per year
+    self.managementDebt = 0
```

Comment:
Do not change default management fee

Reply:
in order to keep the managementFee. 
If it's not enough this time, it will be deducted next time